instances/igrins2/src/main/config/services/edu.gemini.aspen.gmp.services.properties.SimplePropertyHolder-default.cfg

```diff
@@ -0,0 +1,11 @@
+#    Contains the description of the properties that can be obtained through the
+#    GIAPI. Each property has a string value associated.
+
+GMP_HOST_NAME = localhost
+DHS_ANCILLARY_DATA_PATH = /tmp
+DHS_SCIENCE_DATA_PATH = /home/igrins2/data/observeData
+#DHS_PERMANENT_SCIENCE_DATA_PATH = /home/igrins2/data/observeDataPerm
```

Comment:
This is going to be the final config but during the initial phases we'll likely not mount perm until the files are validated, using a local dir instead